kroxylicious/src/main/java/io/kroxylicious/proxy/config/admin/PrometheusMetricsConfig.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy.config.admin;
+
+public class PrometheusMetricsConfig {
+    private final String path;
+
+    public PrometheusMetricsConfig(String path) {
```

Comment:
Does this mean the `path` is configurable? If so why would that be desirable?

Reply:
As in the other answer, users might want to choose between a more descriptive path like `/metrics/prometheus` or the prometheus default of `/metrics`

Reply:
yep, could do that if/when we add a second route.

Reply:
I think it would be a mistake to make paths configurable. The situation with metrics in Strimzi provides a good illustration about why: You no longer have an API that people can actually reliably write clients against. The client ends up needing to _also_ be parameterised for where to find particular things (metrics in the Strimzi example, endpoints here). The end result is a food gun where both server and client side need to be configured (and kept in sync), and the net benefit of all that configurability is low.

Reply:
True, I've been thinking of it a bit more like a framework, but this aspect of kroxy doesn't need the flexibility. It's also easier to start inflexible and make it flexible in future than go in the other direction.

I've made it non-configurable and defaulted to `/metrics`.